bin/blobstreamx.rs

```diff
@@ -48,10 +54,31 @@ impl BlobstreamXOperator {
 
         let succinct_rpc_url = env::var("SUCCINCT_RPC_URL").expect("SUCCINCT_RPC_URL must be set");
         let succinct_api_key = env::var("SUCCINCT_API_KEY").expect("SUCCINCT_API_KEY must be set");
-        let client = SuccinctClient::new(succinct_rpc_url, succinct_api_key);
+
+        let private_key: Option<String>;
```

Comment:
Oh yeah keeping these around as `Option` is why I thought reading from the .env within the actual method would be cleaner lol

Reply:
Yeah, but I think this is more correct if people specify their `PRIVATE_KEY` differently.